Description: Use the system python-chardet and python-urllib3 instead of the
 embedded copies but provide requests.packages package because it will be
 used to supply a stub for ``requests.packages.urllib3``.
Author: Daniele Tricoli <eriol@mornie.org>
Forwarded: not-needed
Last-Update: 2014-10-21

--- a/requests/compat.py
+++ b/requests/compat.py
@@ -4,7 +4,7 @@
 pythoncompat
 """
 
-from .packages import chardet
+import chardet
 
 import sys
 
--- a/setup.py
+++ b/setup.py
@@ -19,7 +19,6 @@
 packages = [
     'requests',
     'requests.packages',
-    'requests.packages.chardet',
     'requests.packages.urllib3',
     'requests.packages.urllib3.packages',
     'requests.packages.urllib3.contrib',

